python/styleguide.md

```diff
@@ -0,0 +1,108 @@
+# Python Code Style Guide
+
+
+Documentation
+-------------
+
+* We use doc strings for package, module, class and function documentation.
+    * A doc string is a string that is the first statement in a package, module, class or function.
+    * These strings can be extracted automatically through the __doc__ member of the object. 
+
+* We always use the three double-quote `"""` format for doc strings.
+    * This is defined in [PEP 257](https://www.python.org/dev/peps/pep-0257/)
+
+* A doc string should be organized as
+    * a summary line (one physical line),
+    * a blank line,
+    * the rest of the doc string.
+
+### On Block comments
+
+* Comment on complicated parts of the code
+    * If you're going to have to explain it at the next code review, you should comment it now.
+
+* Don't describe the code.
```

Comment:
... Don't describe the code and avoid superfluous comments. Generally it's best to convey that information *in the actual code*, for example by using appropriate naming:

Harmful:

```python
d = 0  # Elapsed time in days
```

Better:

```python
elapsed_time_in_days = 0
# or
days_elapsed = 0
```